tests/Unit/Core/Module/SourceHandler/ZipSourceHandlerTest.php

```diff
@@ -36,7 +36,7 @@
 
 class ZipSourceHandlerTest extends TestCase
 {
-    private const TEST_MODULES_DIRECTORY = __DIR__ . '/../../../Resources/modules';
+    private const TEST_MODULES_DIRECTORY = __DIR__ . '/../../../Resources/modules/zip_source_handler.tmp';
```

Comment:
It is not a directory anymore 😛 so the constant name must be changed or the logic must be changed

Reply:
Why do you say it is not a directory ? :thinking: 
Maybe you don't like `.tmp` on a directory name ? Would you prefer `zip_source_handler_tmp` ? Or `tmp_zip_source_handler` ?

Reply:
Or Maybe `zip_source_handler_test`